src/uct/base/uct_iface.h

```diff
@@ -7,6 +7,10 @@
 #ifndef UCT_IFACE_H_
 #define UCT_IFACE_H_
 
+#ifdef HAVE_CONFIG_H
```

Comment:
why needed?

Reply:
> why needed?

below used macro ```IBV_HW_TM``` but config.h is not included. unrelated, removed for now, will create separate PR